importexport/capella/capella.cpp

```diff
@@ -206,11 +206,15 @@ static void processBasicDrawObj(QList<BasicDrawObj*> objects, Segment* s, int tr
                                           case 'u':   // fermata up
                                                 addArticulationText(score, cr, track, SymId::fermataAbove);
                                                 break;
+                                          case 'n':   // segno coda
+                                                addArticulationText(score, cr, track, SymId::coda);
+                                                break;
+                                          case 'y':   // segno
+                                                addArticulationText(score, cr, track, SymId::segno);
+                                                break;
                                           case 'd':   // da capo D.C.
                                           case 'e':   // dal segno D.S.
```

Comment:
those 2 would by `Jump`s

Reply:
Actually I don't see where you deal with D.S. (al Coda/Fine) or D.C. (al Coda/Fine), nor To Coda and Fine

Reply:
D.S. al code is inside the ```<barline> <drawObjects>... </barline>``` tag, so that's why I've added a new member ```CapExplicitBarline::QList<BasicDrawObj*>``` line 146 capella.h. And then I'm initializing it at ```CapExplicitBarline::readCapx(XmlReader& e)``` line 175 capxml.cpp.